src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -71,6 +71,8 @@
 
     private static final Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.IDENTITY).create();
 
+    private final FactBuilder factBuilder = new FactBuilder();
```

Comment:
could be defined per instance and does not have to be passed as parameter between methods